### ISSUE TITLE
[1.10-rc2] Error initializing network controller: could not delete the default bridge network: network bridge has active endpoints

### ISSUE THREAD

**Issue by ibuildthecloud** [docker/docker#19988]
I just got this error and now my 1.10-rc2 daemon won't start:

```
FATA[0001] Error starting daemon: Error initializing network controller: could not delete the default bridge network: network bridge has active endpoints 
```

This was encountered when doing a hard kill on the docker daemon with about 100 containers running on the default bridge network.

**Comment by ibuildthecloud** [comment]
Here's a bit more info from the logs

```
DEBU[0000] Registering ipam driver: "default"           
ERRO[0001] getNetworkFromStore for nid d5eece51105d49cce8874265e1ba30f2c3a9c9b42443f6edcd9a700c09889b2b failed while trying to build sandbox for cleanup: network d5eece51105d49cce8874265e1ba30f2c3a9c9b42443f6edcd9a700c09889b2b not found 
ERRO[0001] getEndpointFromStore for eid 381b95e2725b5812607f4eb1e7d9a5a3c9a790b5e968fe9f6507d581238f3755 failed while trying to build sandbox for cleanup: could not find endpoint 381b95e2725b5812607f4eb1e7d9a5a3c9a790b5e968fe9f6507d581238f3755: [] 
WARN[0001] Failed getting network for ep c13dd53a0b2391818c85e42f26983a3bb94197613fbea226912ffe37b38bbfbd during sandbox de5b822a465563c1b512c822633e98750776efa5579325392f51db1e54dcc1c9 delete: network d5eece51105d49cce8874265e1ba30f2c3a9c9b42443f6edcd9a700c09889b2b not found 
WARN[0001] Failed deleting endpoint 381b95e2725b5812607f4eb1e7d9a5a3c9a790b5e968fe9f6507d581238f3755: failed to get endpoint from store during Delete: could not find endpoint 381b95e2725b5812607f4eb1e7d9a5a3c9a790b5e968fe9f6507d581238f3755: []
 
ERRO[0001] failed to delete sandbox de5b822a465563c1b512c822633e98750776efa5579325392f51db1e54dcc1c9 while trying to cleanup: could not cleanup all the endpoints in container d4ad4ae2fcb3d374f39c181106e1734ce84f8a65aae7107fde1c7af9acc46b0b / sandbox de5b822a465563c1b512c822633e98750776efa5579325392f51db1e54dcc1c9 
DEBU[0001] Cleaning up old shm/mqueue mounts: start.    
FATA[0001] Error starting daemon: Error initializing network controller: could not delete the default bridge network: network bridge has active endpoints 
```

**Comment by mavenugo** [comment]
there are `Network not found` and `could not find endpoint` errors during the cleanup process (due to unclean shutdown) that happens in bootup. But, the container d4ad4ae2fcb3d374f39c181106e1734ce84f8a65aae7107fde1c7af9acc46b0b seem to be refering to this network and endpoint. Since you confirmed that all the containers belong to the default network, I would assume that the above missing network is that of default network (docker0). Do you know why the network and store be missing from the store ?

**Comment by thaJeztah** [comment]
Adding this to the 1.10.1 milestone; we'll investigate further after the release

**Comment by tiborvass** [comment]
Ping @ibuildthecloud

**Comment by tiborvass** [comment]
Ping @ibuildthecloud

**Comment by tiborvass** [comment]
Closing. Might be already fixed. Feel free to comment when you have new info.

**Comment by davidkarlsen** [comment]
I get the same when ipv6 is enabled:
```
root@main:~# docker version
Client:
 Version:      1.10.3
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   20f81dd
 Built:        Thu Mar 10 15:54:52 2016
 OS/Arch:      linux/amd64
An error occurred trying to connect: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.22/version: read unix @->/var/run/docker.sock: read: connection reset by peer
```

**Comment by matheusgontijo** [comment]
Yes, got the same guys. Any clue?

**Comment by thaJeztah** [comment]
@matheusgontijo "got the same" as the original issue, or problem connecting to `/var/run/docker.sock` ? What version of docker?  If you are on Docker 1.13, please check if it's https://github.com/docker/docker/issues/30295; check if your name server has an `%<iface>` in it's address;  see https://github.com/docker/docker/issues/30295#issuecomment-273983364 (and https://github.com/docker/docker/issues/30295#issuecomment-275315635). If it's the same; this issue has been fixed and will be in the docker 1.13.1 release.

Otherwise, please open a _new_ issue with details - commenting issues that were closed over a year a go is not the best way to report bugs, they may go unnoticed and can be unrelated to the issue you're commenting on

**Comment by matheusgontijo** [comment]
Thanks @thaJeztah for the reply. Yes, I already fixed my issue. Everything I had to do was to install a previous version "1.12.0" of Docker and everything worked fine.
